### PR TITLE
Update SIX integration doc to match Odoo 15.0

### PR DESCRIPTION
**Pull request by TwioTech**
My colleague and I spent quite some time before we found out that the SIX integration works well. I hope this change in the documentation will help a lot of people. Thanks!

### CODE REVIEW

**Review by LoredanaLrpz — APPROVED** [review]
Thank you for your contribution, 
I took the liberty to make some changes 

Have a nice day

**Review by jcs-odoo — APPROVED** [review]

File: content/applications/sales/point_of_sale/payment/six.rst [hunk 1]
```diff
@@ -24,11 +24,11 @@ remove the "Apps" filter and search for "POS Six".
     :align: center
 
 Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`,
-Create a new payment method for SIX, select the payment terminal option
-"SIX without IoT Box", and enter your payment terminal IP address.
+Create a new payment method for SIX, select the appropriate journal, select *SIX* in the **Use a Payment Terminal**
+field, then enter your payment terminal IP address.
```
Comment (AntoineVDV) [inline content/applications/sales/point_of_sale/payment/six.rst:28]: - Create -> create
- Don't start a new line unless you've reached the 100th character.
- Start a new line before you reach the 100th character.

```rst
Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
method for SIX, select the appropriate journal, select *SIX* in the **Use a Payment Terminal**
field, then enter your payment terminal IP address.
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
I guess that it's better if you handle this change yourself @LoredanaLrpz :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**